rep-I0001.rst

```diff
@@ -109,43 +109,46 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
         effort_desired[]
         effort_error[]
     
     
 Dynamic Group Status
----------
-The dynamic group status is meant to mimic both the ROS-I RobotStatus message.  See the RobotStatus message[#rbt_stat]_ for field descriptions.::
+--------------------
+The dynamic group status is meant to mimic both the ROS-I RobotStatus message.  See the RobotStatus message [#rbt_stat]_ for field descriptions.::
```

Comment:
"meant to mimic both the ROS-I RobotStatus message": is something missing here? Mimic both RobotStatus and what other message?